cms/envs/common.py

```diff
@@ -1495,6 +1495,7 @@
     # Management of per-user schedules
     'openedx.core.djangoapps.schedules',
     'rest_framework_jwt',
+    'git_auto_export',
```

Comment:
common will work for `devstack` only, please add it in lms.envs.production & cms.envs.production

Reply:
no, `production.py` import common https://github.com/mitocw/edx-platform/blob/master/cms/envs/production.py#L23